src/overlays/actors/ovl_End_Title/z_end_title.c

```diff
@@ -27,13 +27,134 @@ const ActorInit End_Title_InitVars = {
     (ActorFunc)EndTitle_Update,
     (ActorFunc)EndTitle_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_End_Title/EndTitle_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_End_Title/EndTitle_Destroy.s")
+#include "z_end_title_gfx.c"
+
+void EndTitle_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EndTitle* this = THIS;
+
+    this->theEndTitleCardAlpha = 0;
+    this->zeldaTitleCardAlpha = 0;
+    this->ocarinaTitleCardAlpha = 0;
+    if (this->actor.params == 1) {
+        this->actor.draw = func_80B65DA8;
+    }
+}
+
+void EndTitle_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
+
+void EndTitle_Update(Actor* thisx, GlobalContext* globalCtx) {
+}
+
+void EndTitle_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    MtxF* mf;
+    EndTitle* this = THIS;
+    s32 frameCount = globalCtx->csCtx.frames;
+    Player* player = PLAYER;
+    mf = &player->mf_9E0;
+    {
+        GraphicsContext* gfxCtx;
+        Gfx* dispRefs[4];
+        gfxCtx = globalCtx->state.gfxCtx;
+        Graph_OpenDisps(dispRefs, globalCtx->state.gfxCtx, "../z_end_title.c", 403);
+        func_80093D84(globalCtx->state.gfxCtx);
+        Matrix_Mult(mf, MTXMODE_NEW);
+        Matrix_Translate(0.0f, 150.0f, 170.0f, MTXMODE_APPLY);
+        Matrix_Scale(0.13f, 0.13f, 0.13f, MTXMODE_APPLY);
+        Matrix_RotateX(0.287621408700943f, MTXMODE_APPLY);
+        Matrix_RotateY(0.0f, MTXMODE_APPLY);
+        Matrix_RotateZ(0.0f, MTXMODE_APPLY);
+        gSPMatrix(gfxCtx->polyXlu.p++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_end_title.c", 412), G_MTX_LOAD);
+        gSPDisplayList(gfxCtx->polyXlu.p++, triforceDL);
+        Graph_CloseDisps(dispRefs, globalCtx->state.gfxCtx, "../z_end_title.c", 417);
+    }
+
+    {
+        GraphicsContext* gfxCtx;
+        Gfx* dispRefs[4];
+        gfxCtx = globalCtx->state.gfxCtx;
+        Graph_OpenDisps(dispRefs, globalCtx->state.gfxCtx, "../z_end_title.c", 419);
+
+        if ((frameCount >= 891) && (this->theEndTitleCardAlpha < 200)) {
+            this->theEndTitleCardAlpha += 7;
+        }
+        if ((frameCount >= 811) && (this->zeldaTitleCardAlpha < 200)) {
+            this->zeldaTitleCardAlpha += 15;
+        }
+        if ((frameCount >= 851) && (this->ocarinaTitleCardAlpha < 200)) {
+            this->ocarinaTitleCardAlpha += 15;
+        }
+        gfxCtx->overlay.p = func_80093F34(gfxCtx->overlay.p);
+        gDPSetTextureLUT(gfxCtx->overlay.p++, G_TT_NONE);
+        gDPSetEnvColor(gfxCtx->overlay.p++, 255, 120, 30, 0);
```

Comment:
as of now most of us are leaving colors in hex and I think we should leave that consistent for now